tpgtools/api/monitoring/samples/basic.monitored_project.json

```diff
@@ -1,4 +1,4 @@
 {
-  "name": "{{ref:basic.cloudresourcemanager.project.json:name}}",
```

Comment:
This should match https://github.com/GoogleCloudPlatform/declarative-resource-client-library/blob/main/services/google/monitoring/samples/basic.monitored_project.json exactly (same for other cloudresourcemanager references)

Reply:
Problem is, they refer to different things.  This is one of the places we were guessing at correlations between two things and sometimes failing.  The service name in the DCL samples refers to the DCL package, as in the import string, but in tpgtools samples it refers to the service for the resource.

How crucial is it that these are identical?  I can put back the bad guessing logic (slash try to write better guessing logic), or write an importer tool.  Honestly an importer tool might be nice even if I do write the guessing logic, because there are already a good handful of secret steps to getting a sample imported into tpgtools...

Reply:
We want to pull the samples directly from the DCL eventually, same as we did for resource specs, but the need for things like exclusions like w/ multicloud or for resources that depend on non-DCL-based resources means that it's not, like, immediately 5 minutes of work. Given that, I'd consider aligning them exactly is a requirement.

re: secret steps, if they're not written down at https://github.com/GoogleCloudPlatform/magic-modules/tree/master/tpgtools#adding-samples could you write them down there or resolve them so they're no longer needed if you have the cycles?

Reply:
Sure thing, I'll come back with that soon.